flink-table/flink-table-common/src/main/java/org/apache/flink/table/api/constraints/Constraint.java

```diff
@@ -61,6 +61,6 @@
 	 */
 	enum ConstraintType {
 		PRIMARY_KEY,
-		UNIQUE_KEY
+		UNIQUE
```

Comment:
Could you revert this? We can discuss this and change it when we support UNIQUE..